ledger-core/ledger/server/datawriter/smplash.go

```diff
@@ -86,13 +89,19 @@ func (p *SMStreamDropBuilder) stepInit(ctx smachine.InitializationContext) smach
 // 	return prevPN
 // }
 
-func (p *SMStreamDropBuilder) stepCreateStreamDrop(ctx smachine.ExecutionContext) smachine.StateUpdate {
+func (p *SMPlash) stepCreatePlush(ctx smachine.ExecutionContext) smachine.StateUpdate {
+
+	var (
+		tree jet.Tree
+		pop census.OnlinePopulation
```

Comment:
I think that this is too tightly coupled with census. There should be something like API between those, but it is already "gcpv2/api/census", it looks like those layers are mixed up

Reply:
census is separate from gcpv2 - it was only created as a part of it and can be moved out. I don't touch its location just for sake of smaller PR